v1/assets/src/js/pages/revenue-generator-paywall.js

```
@@ -2642,88 +2643,95 @@ import { RevGenModal } from '../utils/rev-gen-modal';
 				merchantId,
 				merchantKey
 			) {
-				if ( ! $o.requestSent ) {
-					$o.requestSent = true;
-
-					// Create form data.
-					const formData = {
-						action: 'rg_verify_account_credentials',
-						merchant_id: merchantId,
-						merchant_key: merchantKey,
-						security:
-							revenueGeneratorGlobalOptions.rg_paywall_nonce,
-					};
-
-					let eventLabel = '';
+				return new Promise( ( response ) => {
```

I'm nitpicking a little bit, but the Promise's first argument is called resolve 😄

---

@tomislavhren Nitpick away! I love feedback. 😄 So `resolve( value )` instead of `response( value )`?

---

Correct! I mean, it's just a naming convention. You could've named it `abc`, but when you make a Promise, you're expecting it to `resolve` or `reject` (the second argument - if you invoke reject, it will throw an error to calling function rather than returning)

---

Got it. That makes perfect sense. Thank you for comprehensive explanation. 🙏